src/measure_theory/integral/lebesgue.lean

```diff
@@ -2240,6 +2240,110 @@ begin
   exact funext (λ a, lintegral_dirac a f),
 end
 
+-- Q: Is this instance a bad idea? It is certainly always the correct one, but does it
+-- affect performance?
+@[priority 100]
+instance measurable_space.top.measurable_singleton_class {α : Type*} :
```

Comment:
Hmm... There is not an error in `src/probability/process/filtration.lean`. I wonder if this instance I added is to blame.

Reply:
```suggestion
instance _root_.measurable_space.top.measurable_singleton_class {α : Type*} :
```
I think this will fix the error